src/app/qgsrasterlayerproperties.cpp

```diff
@@ -1013,6 +1033,15 @@ void QgsRasterLayerProperties::apply()
     mMetadataFilled = false;
   mRasterLayer->setLegendUrlFormat( mLayerLegendUrlFormatComboBox->currentText() );
 
+  QString wmsPrintLayer = mWMSPrintLayerLineEdit->text();
```

Comment:
const

Reply:
(minor comment) By the way, `wmsPrintLayer` seems to be used just once in the `if` condition. I'm not sure it's even necessary to create a variable just for that.

Reply:
Do you mean
'const QString wmsPrintLayer = ....' 
?

Reply:
> const QString wmsPrintLayer = ....'

Yes. 

However, if `wmsPrintLayer` is just used in the `if` condition, then `if ( mWMSPrintLayerLineEdit->text().isEmpty() ) {}` seems enough. But it's just a tiny detail :).